sync/class.jetpack-sync-module-menus.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+class Jetpack_Sync_Module_Menus extends Jetpack_Sync_Module {
+	private $nav_items_just_added = array();
+
+	function name() {
+		return 'menus';
+	}
+
+	public function init_listeners( $callable ) {
+		add_action( 'wp_create_nav_menu', $callable, 10, 2 );
+		add_action( 'wp_update_nav_menu', array( $this, 'update_nav_menu' ), 10, 2 );
+		add_action( 'wp_add_nav_menu_item', array( $this, 'update_nav_menu_add_item' ), 10, 3 );
+		add_action( 'wp_update_nav_menu_item', array( $this, 'update_nav_menu_update_item' ), 10, 3 );
+		add_action( 'post_updated', array( $this, 'remove_just_added_menu_item' ), 10, 2 );
+
+		add_action( 'jetpack_sync_updated_nav_menu', $callable, 10, 2 );
+		add_action( 'jetpack_sync_updated_nav_menu_add_item', $callable, 10, 4 );
+		add_action( 'jetpack_sync_updated_nav_menu_update_item', $callable, 10, 4 );
+		add_action( 'delete_nav_menu', $callable, 10, 3 );
+
+	}
+	
+	public function update_nav_menu( $menu_id, $menu_data = array() ) {
+		if ( empty( $menu_data ) ) {
+			return;
+		}
+		/**
+		 * Helps sync log that a nav menu was updated.
+		 *
+		 * Helps Sync log that a got cleared from inactive.
```

Comment:
Actually, do you mean to have this line in the comment at all since it is beneath one that says the call is to "Helps sync log that a nav menu was updated."?